AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1740,6 +1740,7 @@ pcmag.com###article + div[class] > div[x-data="window.newsletters()"]
 makeuseof.com,fosspost.org,sergeiyurev.com##.ml-subscribe-form
 skyeng.ru##.email-subscription
 clara.es##.content-newsletter
+ru.vpnmentor.com##.content-wrap.article-thanks-wrap
```

Comment:
Exchange this rule to the `vpnmentor.com##.main-content-holder > div.article-thanks-wrap` and move it to the AdGuard Base filter.
Also block the ad `vpnmentor.com##.side-top-vendors-wrap`.